libs/tup-components/src/tickets/TicketsTable.tsx

```diff
@@ -93,7 +93,10 @@ export const TicketsTable: React.FC<{ ticketsPath: string }> = ({
             >
               <td>{ticket.numerical_id}</td>
               <td>
-                <Link to={`${ticketsPath}/${ticket.numerical_id}`}>
+                <Link
+                  to={`${ticketsPath}/${ticket.numerical_id}`}
+                  state={{ fromLink: true }}
```

Comment:
The `fromLink` "state" is only used if a component loaded by the `to` path knows how to interpret it, correct? Which is like ["state" in `Navigate`](https://github.com/TACC/tup-ui/blob/v1.0.13/libs/tup-components/src/utils/RequireAuth.tsx#L18), correct?

Reply:
Yeah, the "state" in `<Link/>` and the "state: in `<Navigate/>` call the same API under the hood I believe. In this case we're using it to mark that navigation occurred from inside the portal so we can safely use the browser's "back" behavior. Otherwise if I was on a random website and pasted the URL for a specific ticket, closing the modal would actually navigate my browser back to the starting point.